### ISSUE TITLE
Icon fehlt in iCloud Settings

### ISSUE THREAD

**Issue by F3000** [remuslazar/EVPlugFinderPublic#103]
Eben per zufall festgestellt. 
Zu finden unter : 

Einstellungen -> Apple ID -> iCloud -> Speicher verwalten -> vor EVPlugFinder ist nur das Apple standard Icon zu sehen . 

Vielleicht kann man das fixen .

[img_0175](https://user-images.githubusercontent.com/8161283/34395729-20dab13e-eb63-11e7-9274-dba1cfc7d451.jpg)

**Comment by remuslazar** [comment]
@F3000 danke für den Hinweis, das konnte ich reproduzieren.

**Comment by F3000** [comment]
Alles klar :) 


Aber nichtmal Apple schafft es ihre Icons richtig zu machen.

**Comment by remuslazar** [comment]
Ja, das ist auch gar nicht so einfach, weil der Sprecher i.d.R. nicht der App gehört sonder der sog. "App Group".

Siehe auch https://forums.developer.apple.com/thread/52090
